### ISSUE TITLE
Exception when launching operation

### ISSUE THREAD

**Issue by clong** [mitre/caldera#14]
Repro steps:
1. Enrolled 3 hosts with cagent and confirmed they show up in the debug console
2. Created a network that includes all 3 hosts
3. Created an operation that bootstraps a rat, is given hardcoded credentials, and begins the operation on a single host

The python exception I receive (multiple times) is as follows:
```
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/aiohttp/helpers.py", line 554, in __get__
    return inst._cache[self.name]
KeyError: 'remote'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/aiohttp/helpers.py", line 521, in log
    for key, value in fmt_info:
  File "/usr/local/lib/python3.5/dist-packages/aiohttp/helpers.py", line 513, in <genexpr>
    for key, method in self._methods)
  File "/usr/local/lib/python3.5/dist-packages/aiohttp/helpers.py", line 473, in _format_a
    ip = request.remote
  File "/usr/local/lib/python3.5/dist-packages/aiohttp/helpers.py", line 556, in __get__
    val = self.wrapped(inst)
  File "/usr/local/lib/python3.5/dist-packages/aiohttp/web_request.py", line 321, in remote
    peername = transport.get_extra_info('peername')
  File "/usr/lib/python3.5/asyncio/sslproto.py", line 306, in get_extra_info
    return self._ssl_protocol._get_extra_info(name, default)
  File "/usr/lib/python3.5/asyncio/sslproto.py", line 537, in _get_extra_info
    return self._transport.get_extra_info(name, default)
AttributeError: 'NoneType' object has no attribute 'get_extra_info'
```

This may be related to me using a different version of aiohttp detailed in this PR: https://github.com/mitre/caldera/pull/13

**Comment by clong** [comment]
Manually applying this patch seems to have taken care of the issue: https://github.com/python/cpython/commit/2b27e2e6a35c3d3e369612b984017fe0d1bfcf0c

Looks like it's not available except for Python 3.7 according to the tags:
![image](https://user-images.githubusercontent.com/5092699/35025071-20e7e922-faf8-11e7-903f-4c8206ab9fd6.png)

**Comment by dm-mitre** [comment]
I think you're right on the root cause. What version of Python were you running? I think this fix was rolled into [3.5.4](https://docs.python.org/3.5/whatsnew/changelog.html#python-3-5-4) and [3.6.2](https://docs.python.org/3.6/whatsnew/changelog.html#python-3-6-2-final)

**Comment by clong** [comment]
Ah, I'm only running 3.5.2. I'll upgrade and see if that takes care of it - thanks!

**Comment by dm-mitre** [comment]
@clong did the upgrade fix things?

**Comment by clong** [comment]
@dm-mitre Sorry, haven't had time to test yet. Will probably be a few more days if that's okay. Looks like the only reasonable way to get 3.5.4+ on 16.04 is to compile from source.

**Comment by dm-mitre** [comment]
no worries. FWIW I think the exception is purely cosmetic, I don't think functionality changes at all, just clogs up the output messages

**Comment by clong** [comment]
This is fixed using Python 3.6.4
